pkg/asset/installconfig/powervs/session.go

```diff
@@ -22,14 +21,6 @@ var (
 // Session is an object representing a session for the IBM Power VS API.
 type Session struct {
 	Session *ibmpisession.IBMPISession
```

Comment:
did you try using this directly instead of wrapping it inside a struct? just curius

Reply:
No, I can look into that after this fix if you think that it would be benificial

Reply:
Not a priority but if you remember while you're doing something else I wouldn't mind